### ISSUE TITLE
Dragging a youtube or vimeo video outside the editor results in an exception in IE11

### ISSUE THREAD

**Issue by starquake** [tinymce/tinymce#2953]
It can even sometimes crash the browser (I think it sometimes ends up in a loop)

Steps to reproduce:

- Open http://www.tinymce.com in IE11
- Remove contents of the example editor
- Add a youtube movie using "Insert" -> "Insert/edit video"
- Make sure the video is not focused (you can see a blinking cursor next to it)
- Drag the video outside of the editor

The exception is triggered by findClosestIeRange.

Catching the error like this does not resolve the crashing issue:

```

RangeUtils.getCaretRangeFromPoint = function(clientX, clientY, doc) {
		var rng, point;

		if (doc.caretPositionFromPoint) {
			point = doc.caretPositionFromPoint(clientX, clientY);
			rng = doc.createRange();
			rng.setStart(point.offsetNode, point.offset);
			rng.collapse(true);
		} else if (doc.caretRangeFromPoint) {
			rng = doc.caretRangeFromPoint(clientX, clientY);
		} else if (doc.body.createTextRange) {
			rng = doc.body.createTextRange();

			try {
				rng.moveToPoint(clientX, clientY);
				rng.collapse(true);
			} catch (ex) {
			    try {
			        rng = findClosestIeRange(clientX, clientY, doc);
			    } catch (ex2) {
                                // Now we're really out of options...
			    }
			}
		}

		return rng;
	};

```

**Comment by blackcathikari** [comment]
Hi, we've investigated this issue and cannot reproduce it in TinyMCE v4.4.0. Thanks, the TinyMCE Team.

**Comment by starquake** [comment]
This issue has not been fixed. What happens right now is the entire video disappears when dragging! I will open a new issue.

**Comment by dsakko** [comment]
Hi starquake. The original problem of the exception / browser crashing should now be fixed in 4.4. I've looked into it and following your steps it no longer triggers the exception. 

Thanks for adding the new bug. It's now showing different behaviour and we'll track our investigations in that bug.
